### PR TITLE
Feature/get purchasedashboard views tests

### PR DESCRIPTION
**Pull request by Lex-23**

### CODE REVIEW

**Comment by Lex-23** [comment]
[Write logic for BUY ASSETS from market](https://trello.com/c/nzNYUyZA/16-write-logic-for-buy-assets-from-market)

File: cryptostock/account/tests/test_get_list_purchasedashboard.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+
+def test_get_list_purchasedashboard_db_calls(auth_broker, broker_account):
+    PurchaseDashboardFactory.create_batch(10, broker=broker_account)
+
+    with CaptureQueriesContext(connection) as query_context:
+        response = auth_broker.get("/api/purchasedashboard/")
+
+    assert response.status_code == 200
+    assert len(query_context) == 6
```
Comment (vadim-denisov) [inline cryptostock/account/tests/test_get_list_purchasedashboard.py:90]: Can we reduce db calls?

Comment (Lex-23) [reply cryptostock/account/tests/test_get_list_purchasedashboard.py:90]: Already reduced maximum

Comment (Lex-23) [reply cryptostock/account/tests/test_get_list_purchasedashboard.py:90]: 54 >> 6

Comment (Lex-23) [reply cryptostock/account/tests/test_get_list_purchasedashboard.py]: Now is 5

**Review by vadim-denisov — APPROVED** [review]

**Pull request merged**